To: vim_dev@googlegroups.com
Subject: Patch 7.3.330
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.330
Problem:    When longjmp() is invoked if the X server gives an error the state
	    is not properly restored.
Solution:   Reset vgetc_busy. (Yukihiro Nakadaira)
Files:	    src/main.c


*** ../vim-7.3.329/src/main.c	2011-05-10 16:41:13.000000000 +0200
--- src/main.c	2011-10-04 16:41:03.000000000 +0200
***************
*** 1009,1014 ****
--- 1009,1015 ----
  	skip_redraw = FALSE;
  	RedrawingDisabled = 0;
  	no_wait_return = 0;
+ 	vgetc_busy = 0;
  # ifdef FEAT_EVAL
  	emsg_skip = 0;
  # endif
*** ../vim-7.3.329/src/version.c	2011-09-30 18:35:49.000000000 +0200
--- src/version.c	2011-10-04 16:42:23.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     330,
  /**/

-- 
Just remember...if the world didn't suck, we'd all fall off.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
